modules/account-lib/src/coin/dot/transaction.ts

```diff
@@ -0,0 +1,193 @@
+import { BaseCoin as CoinConfig } from '@bitgo/statics';
+import { BaseTransaction, TransactionType } from '../baseCoin';
+import { BaseKey } from '../baseCoin/iface';
+import { InvalidTransactionError, SigningError } from '../baseCoin/errors';
+import { construct, decode } from '@substrate/txwrapper-polkadot';
+import { UnsignedTransaction } from '@substrate/txwrapper-core';
+import { TypeRegistry } from '@substrate/txwrapper-core/lib/types';
+import { decodeAddress } from '@polkadot/keyring';
+import { KeyPair } from './keyPair';
+import { TxData, DecodedTx, TransferArgs, StakeArgs, StakeArgsPayeeRaw, AddProxyArgs, ProxyArgs } from './iface';
+import utils from './utils';
+
+export class Transaction extends BaseTransaction {
```

Comment:
An estimate size method could be usefull for the fee calculation on the tx.
https://wiki.polkadot.network/docs/learn-transaction-fees#fee-calculation

Reply:
okay yes we can add this @nimraarif21